code/game/machinery/computer/cloning.dm

```diff
@@ -150,7 +150,7 @@
 	var/list/temprecords[0]
 	for(var/datum/dna2/record/R in records)
 		var tempRealName = R.dna.real_name
-		temprecords.Add(list(list("record" = "\ref[R]", "realname" = tempRealName)))
+		temprecords.Add(list(list("record" = "\ref[R]", "realname" = sanitize(tempRealName))))
```

Comment:
yes, this will show weird HTML for anyone that has quotes in their name
no, I don't care, they should get a better name